master-libvirt/master.cfg

```diff
@@ -154,6 +154,9 @@ def getDebInstallStep():
         env=envFromProperties(['test_mode', 'test_type', 'branch', 'master_branch', 'arch', 'dist_name', 'version_name', 'mariadb_version', 'major_version', 'systemdCapability', 'needsGalera', 'parentbuildername', 'BB_CI']),
         command=['./deb-install.sh'])
 
+def getMajorVersionStep():
+    steps.setProperty('major_version', step.getProperty("mariadb_version").removeprefix('mariadb-').rsplit('.',1)[0])
```

Comment:
`step` is undefined and I think `steps.setProperty` doesn't exist. It should be `steps.SetProperty` (https://docs.buildbot.net/latest/manual/configuration/steps/set_property.html). Also, I think the only way here to get the `mariadb_version` property is using the `Interpolate` function.

Reply:
`utils.py:    step.setProperty` prob needs fixing. Ack on the `step` undefined, needs something more thorough. I'm out of time to fix this until after leave.

Reply:
I can give it a try later today or tomorrow morning.